client-spark/spark3/src/main/java/org/apache/spark/shuffle/writer/RssShuffleWriter.java

```diff
@@ -694,4 +745,14 @@ private void throwFetchFailedIfNecessary(Exception e) {
     }
     throw new RssException(e);
   }
+
+  @VisibleForTesting
+  protected void enableBlockFailSentRetry() {
+    this.isBlockFailSentRetryEnabled = true;
+  }
+
+  @VisibleForTesting
+  protected void addReassignmentShuffleServer(String shuffleId, ShuffleServerInfo replacement) {
+    failoverShuffleServers.put(shuffleId, replacement);
```

Comment:
`faultServers` or `replacementServer`? What's the meaning of `failoverServers`?

Reply:
done